app/models/miq_shortcut.rb

```diff
@@ -20,8 +26,7 @@ def self.seed
       end
     end
 
-    all.each do |rec|
-      next if names.include?(rec.name)
+    db_data.each do |name, rec|
```

Comment:
We only delete the records from the local list.
The records left in `db_data` are the records that did not have a corresponding seed.